networkdb/networkdb.go

```diff
@@ -141,6 +141,9 @@ type network struct {
 
 	// Number of gossip messages sent related to this network during the last stats collection period
 	qMessagesSent int
+
+	// Number of entries on the network
+	entriesNumber int
```

Comment:
I guess this is used only for debugging purposes... It would be good to write a comment reflecting the purpose and we should not be tempted to use this as a reference counter for any logic in the future.

Reply:
will add extra comments